crates/api_models/src/payments.rs

```diff
@@ -4873,6 +4874,34 @@ pub struct PaymentsSessionRequest {
     pub merchant_connector_details: Option<admin::MerchantConnectorDetailsWrap>,
 }
 
+#[derive(Debug, serde::Deserialize, serde::Serialize, Clone, ToSchema)]
+pub struct PaymentsPostSessionTokensRequest {
+    /// The unique identifier for the payment
+    #[serde(skip_deserializing)]
+    #[schema(value_type = String)]
+    pub payment_id: id_type::PaymentId,
```

Comment:
If we're not obtaining this from the request (not deserializing it), why are we including it in the request?

Reply:
This is for ApiEventMetrics and ApiLocking

Reply:
@Narayanbhat166 Should we consider having an "internal" struct for such purposes, or are we okay with this practice?